docs/source/contributing/development.rst

```diff
@@ -10,8 +10,8 @@ For first-time contributors
 
 
 #. Fork the project. Go to https://github.com/ManimCommunity/manim and
-   click the "fork" button to create your own copy of the project. You will
```

Comment:
It kinda is but this is better

Reply:
I think "your own copy" is better than "your copy"... 
Saying just "your copy" presupposes that people understand that "fork" means "make a copy".

One could even say it will "create a copy of the project for you to work on".

Reply:
Ok I changed it to "create a copy of the project for you to work on".